backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java

```diff
@@ -751,14 +711,14 @@ public Project deepCopyProjectForUser(String srcProjectID, UserInfo newOwner)
       String workspaceId,
       WorkspaceType workspaceType,
       String projectId,
-      ProjectVisibility projectVisibility) {
+      VisibilityEnum.Visibility visibility) {
     return roleService.getWorkspaceRoleBindings(
         workspaceId,
         workspaceType,
         projectId,
         ModelDBConstants.ROLE_PROJECT_ADMIN,
         ModelDBServiceResourceTypes.PROJECT,
-        projectVisibility.equals(ProjectVisibility.ORG_SCOPED_PUBLIC),
+        visibility.equals(VisibilityEnum.Visibility.ORG_SCOPED_PUBLIC),
```

Comment:
```suggestion
        visibility.equals(ResourceVisibility.ORG_DEFAULT),
```
Due to some miscommunication, we ended up with
ORG_SCOPED_PUBLIC -> ORG_DEFAULT
and
ORG_CUSTOM == ORG_SCOPED_PUBLIC (we should change the proto enum name *very* soon)

Reply:
What is the correct value to be using here?